crates/dslab-mp-python/python/dslabmp.py

```diff
@@ -97,12 +99,20 @@ def __init__(self, t: JSON):
         self.inner = t
 
     def serialize(self):
-        return json.dumps(self.inner)
+        def assertion_serializable(o):
+            assert isinstance(o, Message), 'cannot serialize custom classes' 
+            return o.toJSON()
```

Comment:
Предлагаю убрать `toJSON` из Message, чтобы избежать путаницы:

```suggestion
            return {"_type": self._type, "_data": self._data}
```

Reply:
тоже неактуально